app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Api::V1::CommentsController < ActionController::API
+  def index
+    post = Post.find(params[:post_id])
+    render json: post.comments
+  end
+
+  def create
+    post = Post.find(params[:post_id])
+    comment = post.comments.new(comment_params)
+    comment.author_id = current_user.id
+
+    if comment.save
+      render json: { notice: 'Comment successfully created', comment: }, status: :created
+    else
+      render json: { errors: comment.errors.full_messages }, status: :unprocessable_entity
+    end
```

Comment:
- [ ] Kindly make sure that you have the correct file and folder structure in your Rails application. In a versioned API, you would typically have a directory structure like this:
`app/controllers/api/v1/`
```rb
# Inside the v1 directory, you should have your API controllers and modules.
```
![image](https://github.com/MarcoDiaz2000/blog-app/assets/97100091/28589393-ad4d-4baa-a714-2c108985c4e8)

Reply:
Hi @christianbiring1 
thanks for the comments.

in that order we have it but it seems that when it was uploaded it was changed
![api](https://github.com/MarcoDiaz2000/blog-app/assets/125866339/9902bc43-120a-4628-a492-ff8c640193dd)

Reply:
I will fix it, thanks